File: tests/fixtures/java/sample.java
package com.example.sample;

import java.util.List;
import static java.lang.Math.max;

interface Greeter {
    String greet(String name);
}

class UserService implements Greeter {
    public String greet(String name) {
        return format(name);
    }

    public int clamp(int value) {
        return max(value, 1);
    }

    private String format(String name) {
        return name.trim();
    }
}

File: tests/fixtures/csharp/Sample.cs
using System;
using Acme.Shared;

namespace Acme.Sample;

interface IGreeter
{
    string Greet(string name);
}

class UserService : IGreeter
{
    private const int MinimumLength = 1;

    public string Name { get; set; } = "";

    public string Greet(string name)
    {
        return Format(name);
    }

    private string Format(string name)
    {
        Console.WriteLine(MinimumLength);
        return name.Trim();
    }
}
